lint/src/main/java/com/picover/lint/detector/StringResourceCapitalizationDetector.kt

```diff
@@ -24,8 +24,8 @@ class StringResourceCapitalizationDetector : Detector(), XmlScanner {
 		listOf(SdkConstants.TAG_STRING)
 
 	override fun visitElement(context: XmlContext, element: Element) {
-		val attribute = element.getAttributeNode("name")
-		if (attribute.value.first().isUpperCase().not()) {
+		val attribute = element.getAttributeNode(SdkConstants.ATTR_NAME)
+		if (attribute.value.first().isLowerCase()) {
```

Comment:
Yesterday, you mention about it, this statement checking only the first letter, not the whole `PascalCase` 😄

Reply:
True but note the tile of the PR ;)